bundle.Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=operator-certification-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0+git
```

Comment:
If this is working properly I'd expect to see `v1.15.0` here. I think it's using your local version still.

Reply:
No. That is the version that is displayed when you download the release from github. This is working properly.

Reply:
Are we good here?